Quantum-Control-Applications/NV-Centers/Cryogenic nanophotonic cavity/Use case 1 - Faraon Lab - sub-ns timetagging/beatnotes_test_threshold.py

```diff
@@ -0,0 +1,99 @@
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+from qm import SimulationConfig
+import matplotlib.pyplot as plt
+from configuration import *
+from qualang_tools.results import progress_counter, fetching_tool
+
+import datetime
+import time
+
+###################
+# The QUA program #
+###################
+
+n_avg = 6e6 * 6
+
+resolution = 50  # ps
+t_vec = np.arange(0, meas_len * 1e3, 1)
+
+ttl_wait = 500 // 4
+
+AC_len = 250  # 1 us
+
+
+with program() as calib_delays:
+    times = declare(int, size=100)  # 'size' defines the max number of photons to be counted
+    times_st = declare_stream()  # stream for 'times'
+    counts = declare(int)  # variable to save the total number of photons
+    i = declare(int)  # variable used to save data
+    n = declare(int)  # variable used in for loop for averaging
+    n_st = declare_stream()  # stream for 'iteration'
+
+    with for_(n, 0, n < n_avg, n + 1):
+
+        reset_phase("A_transition")
+        play("laser_ON", "A_transition", duration=AC_len)  # 2 us = 500 clock cycles
```

Comment:
The comment is not consistent with the duration defined above. 
Is this duration important? We just want it to be larger than ttl_wait+40ns no?